### PR TITLE
Re-enable dmd-nightly image

### PR DESCRIPTION
**Pull request by MoonlightSentinel**
Includes a slight change to accomodate for the compatibilty symlink created by dlang/installer#483.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ ENV DLANG_VERSION=$DLANG_VERSION
 ENV DLANG_EXEC=$DLANG_EXEC
 
 # Download and run the install script
-RUN curl -fsS -o /tmp/install.sh https://dlang.org/install.sh
```
Comment (Geod24) [inline Dockerfile]: Why is this necessary ?

File: test.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ trap 'err_report $LINENO' ERR
 grepOutput() {
     local output_file="command_output.tmp"
 
-    if ! "${@:1:(($# - 1))}" > $output_file
+    if ! "${@:1:(($# - 1))}" &> $output_file
```
Comment (Geod24) [inline test.sh:15]: I suppose you tested this ? I would expect this syntax to daemonize the process, not redirect `stderr`.
The syntax I expected was `2>&1 >$output_file`

File: Dockerfile (hunk 1 above)
Comment (MoonlightSentinel) [reply Dockerfile]: dlang/installer#483 added support the new nightlies but is not yet available from dlang.org.
I'll revert this change and see if it still passes with the old nightlies.

File: test.sh (hunk 2 above)
Comment (MoonlightSentinel) [reply test.sh:15]: `&>` captures both `stdout` and `stderr`, see the [bash reference manual](https://www.gnu.org/software/bash/manual/html_node/Redirections.html), 3.6.4.

Comment (MoonlightSentinel) [reply test.sh:15]: Btw, this change is required to fix this test (which checks error messages written to `stderr`):

https://github.com/dlang-tour/core-exec/blob/ac80fa8c937f71788cbe27025419d95af308a706/test.sh#L127-L133

File: Dockerfile (hunk 1 above)
Comment (MoonlightSentinel) [reply Dockerfile]: Didn't work, will need to wait until the new version is published to the homepage...

Comment (wilzbach) [reply Dockerfile]: Deployed the install.sh to dlang.org

Comment (MoonlightSentinel) [reply Dockerfile]: Thanks!

**Review by Geod24 — APPROVED** [review]
Thanks!
